CHANGELOG.md

```diff
@@ -1,6 +1,18 @@
 # Braintree iOS SDK Release Notes
 
 ## unreleased
+* BraintreeVenmo
+  * Add the following properties to `BTVenmoRequest`
```

Comment:
General thought for merchant/sales/support clarity. We should add a line item between the heading and properties about this feature. Something like `Allow merchants to collect enriched customer data if enabled`. This way if someone needs to know when we added this feature it's easy to find in our changelog.